### DISCUSSION TITLE
CW does not start without internet

### DISCUSSION THREAD

**Discussion by StafLoker** [aws/codewhisperer-command-line-discussions#43] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

Run CW

### Actual behaviour

The application does not start if you are not connected to the Internet (an error occurs).

### Environment

```yaml
CODEWHISPERER-details:
  - 0.12.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M2 Pro
  - cores:
  - mem:
os-info:
  - macOS 14.1.2 (23B92)

**Comment by grant0417** [comment]
Can you share the error?

**Reply by StafLoker** [reply]
How do it?

**Comment by brendanfalk** [comment]
@StafLoker Unfortunately right now we need the internet to load up CLI completions the first time. But after that, we cache aggressively and so you shouldn't need the internet again... 

I am going to reopen this issue so other people can upvote if needed. In the meantime, please comment if you have more information to share with us about how to reproduce

**Comment by ohmantics** [comment]
Offline work is always a requirement.

**Reply by grant0417** [reply]
Are you having this as an issue? This should not be a limitation, can you share an error if you have one?

**Reply by ohmantics** [reply]
Nope, just preflighting migration from Fig to CW. I do use my laptop on airplanes with no connectivity.
